Content.Server/Fax/FaxSystem.cs

```diff
@@ -399,6 +442,13 @@ public void Copy(EntityUid uid, FaxMachineComponent? component = null)
         if (sendEntity == null)
             return;
 
+        if (HasComp<MobStateComponent>(sendEntity))
+        {
+            Faxecute(uid, component);
+            return;
+        }
+
+
```

Comment:
Methods should try to be single-purpose but this has the hidden mechanic of killing the entity for any callers when it should be its own method.